setup.py

```diff
@@ -1,26 +1,14 @@
 from setuptools import setup
-from setuptools.extension import Extension
-from setuptools.command.sdist import sdist as _sdist
-
-cmdclass = {}
-
-
-class sdist(_sdist):
-    def run(self):
-        from Cython.Build import cythonize
-        cythonize(['sha256.pyx'])
-        _sdist.run(self)
-cmdclass['sdist'] = sdist
+from Cython.Build import cythonize
 
 
 setup(
     name='sha256',
-    version='0.3',
+    version='0.4',
```

Comment:
I think at this point it would be safe to call it 1.0 now.

Reply:
Done.